galley/docker/testing/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:xenial
```

Comment:
any chance to make multistage from scratch images - or do you want to have base because this is only to be used for testing/debugging ?